xml/System/ArraySegment`1.xml

```diff
@@ -1271,18 +1264,10 @@
       </Parameters>
       <Docs>
         <param name="item">The object to remove from the array segment.</param>
-        <summary>Removes the first occurrence of a specific object from the array segment.</summary>
-        <returns>
-          <see langword="true" /> if <paramref name="item" /> was successfully removed from the array segment; otherwise, <see langword="false" />. This method also returns <see langword="false" /> if <paramref name="item" /> is not found in the array segment.</returns>
-        <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
- This member is an explicit interface member implementation. It can be used only when the <xref:System.ArraySegment%601> instance is cast to an <xref:System.Collections.Generic.ICollection%601> interface.  
-  
- ]]></format>
-        </remarks>
-        <exception cref="T:System.NotSupportedException">The array segment is read-only.</exception>
+        <summary>Throws a <see cref="T:System.NotSupportedException" /> exception in all cases.</summary>
+        <returns>Throws a <see cref="T:System.NotSupportedException" /> exception in all cases.</returns>
+        <remarks>To be added.</remarks>
+        <exception cref="T:System.NotSupportedException">In all cases.</exception>
```

Comment:
Same comment here -- the array segment (and underlying array) cannot be contracted.